python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java

```diff
@@ -452,6 +459,28 @@ private String getPythonCommand() {
     }
   }
 
+  public void setCurrentCondaEnvName(String envName) {
```

Comment:
these conda related funcs can be moved into `PythonCondaInterpreter`?

Reply:
I have considered that.
but, I think it is not good for two reasons.

1. `PythonCondaInterpreter` actually executes only` conda` command line.
It has nothing to do with python.
PythonInterpreter.class is actually affected.
In fact, we can posible uninstall for python via Conda.

2. All derived classes, such as other PythonDockerInteprters, including `PythonCondaInterpreter`, do not have state. I am burdened with keeping the conda configuration information in a PythonCondaInterpreter that does not need to maintain it.

Reply:
@zjffdu Hi, What do you think of?

Reply:
I agree with @1ambda that we should not couple `PythonCondaInterpreter` & `PythonInterpreter` closely. `PythonInterpter` doesn't need to depend on PythonCondaInterpreter, it just depends on `zeppelin.python` IIUC.